runtime/Makefile

```diff
@@ -18,10 +18,18 @@ SOURCES:=$(shell find . -name '*.go')
 GOMOD := $(shell go env GOMOD)
 GOSUM := $(GOMOD:.mod=.sum)
 DOCKER_IMAGE_TAG?=latest
+GO_CACHE_VOLUME_NAME?=gocache
+FIRECRACKER_CONTAINERD_TEST_IMAGE?=localhost/firecracker-containerd-test
 
 INTEG_TEST_SUFFIX := _Isolated
-INTEG_TESTNAMES=$(shell go test -list . | sed '$$d' | grep $(INTEG_TEST_SUFFIX))
-
+INTEG_TESTNAMES=$(shell docker run --rm \
```

Comment:
"make test" needs Go and would download the dependencies of firecracker-containerd. Do we need "docker run" here? Do we want to reduce what developers need to have on the host machine where Docker containers are running?

Reply:
This allows us to reuse the `gocache` volume and, like you were getting at, reduces the places where we require go actually be installed on the host system (I would need to check if it fully eliminates all of them, it might). Unless I'm missing something, `make test` shouldn't be a strict requirement for `go` because there is also the `test-in-docker` step right below it that calls `make test` in a container, which can be used instead.

At worst, this is harmless because the integ-tests need to run in docker containers anyways, so given the small benefits doing it this way is my preference. Let me know what you think.

Reply:
I see. Yeah. I completely forgot about the existence of "make test-in-docker".

Our BuildKite host is currently having Go, but we might remove that to verify all (or most of core) targets work without having Go on the host.